### ISSUE TITLE
Release ansible-core with semver

### ISSUE THREAD

**Issue by felixfontein** [ansible/proposals#192]
# Proposal: use semver for ansible-core

*Author*: Felix Fontein @felixfontein

*Date*: 2021-05-01

- Status: New
- Proposal type: process
- Estimated time to implement: not much

## Motivation
Basically what #179 said.

### Problems
It will avoid the false assumption by some people that 2.x.0 releases do not have breaking changes. Also then ansible-core will use the same versioning scheme that the Ansible package is using, and that's used by most collections.

## Solution proposal
Basically shift the version numbers by one: instead of 2.x.Y, release `X.0.Y` (assuming each major version will not receive new features). Then everyone who understands semver knows what to expect.

## Anything else?
This is basically #179 but targeted at ansible-core, since the Ansible package already did the shift to semver.

**Comment by bcoca** [comment]
Most discussions of #179 would now apply to ansible-core, as of now there are no plans to change the versioning scheme.
